sdk/communication/communication-identity/rollup.patched.config.js

```diff
@@ -0,0 +1,17 @@
+import inject from "@rollup/plugin-inject";
```

Comment:
the rollup changes were introduced as an outcome of the [discussion](https://teams.microsoft.com/l/message/19:344f6b5b36ba414daa15473942c7477b@thread.skype/1673535792000?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1673535792000&teamName=Azure%20SDK&channelName=Language%20%E2%80%93%20JS%E2%80%89%EF%BC%86%E2%80%89TS%20%F0%9F%A5%B7&createdTime=1673535792000) ?

Reply:
Yes, these changes were added by "Deyaaeldeen Almahallawi", so as to run parameterized tests in browser mode.

Reply:
they use it here as well: https://github.com/Azure/azure-sdk-for-js/blob/ef1674336128cbe501978b5865d563b84cc0f16f/sdk/schemaregistry/schema-registry-avro/rollup.test.config.js